src/app/create-event/EventTextarea.tsx

```diff
@@ -0,0 +1,56 @@
+import cx from 'classnames';
+import { FieldErrors } from 'react-hook-form';
+
+import { AriaLiveErrorMessage } from '../../components/AriaLiveErrorMessage';
+
+type InputProps = {
+  register: any;
+  type?: string;
+  label: string;
+  name: string;
+  placeholder?: string;
+  errors: FieldErrors;
+  watchedValue?: string;
+  className?: string;
+};
+
+export const EventTextarea: React.FC<InputProps> = ({
```

Comment:
Why is it EventTextArea? Why not TextArea in some form elements? What it has to do with the events?